src/server/world.zig

```diff
@@ -16,6 +16,7 @@ const Vec3i = vec.Vec3i;
 const Vec3d = vec.Vec3d;
 const Vec3f = vec.Vec3f;
 const terrain = server.terrain;
+const migrations_zig = main.migrations;
```

Comment:
Please just call it `migrations`, there shouldn't be any name conflicts in this file.